Components/LandingFooter.tsx

```diff
@@ -44,16 +44,16 @@ const LandingFooter = () => {
                 <a href="https://github.com/PiyushKalyanpy/GyanaGuru" target="_blank" className={hoverEffect}><AiFillGithub className="mr-1" />Github</a>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                   <a href="https:twitter.com" className={hoverEffect}><AiFillTwitterCircle className="mr-1" />Twitter</a>
+                   <a href="https:twitter.com" className={hoverEffect}><AiFillTwitterCircle className="mr-1 transition ease duration-200 group-hover:text-sky-500" />Twitter</a>
```

Comment:
Why these links are not updated?

@PiyushKalyanpy, I think we can remove these if we don't want to update the correct links here.

Reply:
@lalitmee if you have correct links then provide me otherwise I'll remove them.

Reply:
@lalitmee Any update on what should I do?

Reply:
Let us keep those links for now, we will update them later.